packages/manager/modules/netapp/src/dashboard/volumes/dashboard/acl/template.html

```diff
@@ -17,113 +17,142 @@ <h2>
     >
         <span data-translate="netapp_volumes_acl_add"></span>
     </oui-button>
-    <oui-datagrid rows="$ctrl.acls" ng-if="$ctrl.acls.length">
-        <oui-datagrid-column
-            title=":: 'netapp_volumes_acl_access_type' | translate"
+
+    <table class="oui-table" data-ng-if="$ctrl.acls.length">
```

Comment:
Could the Sonarcloud report be taken in account here please?

Reply:
Yep sure i will take a look